indexer/versionedscanner.go

```diff
@@ -110,8 +110,28 @@ func (vs VersionedScanners) VStoRS() []RepositoryScanner {
 	return out
 }
 
+func (vs *VersionedScanners) FStoVS(scnrs []FileScanner) {
+	n := len(scnrs)
+	if cap(*vs) < n {
+		*vs = make([]VersionedScanner, n)
+	} else {
+		*vs = (*vs)[:n]
+	}
+	for i := 0; i < n; i++ {
+		(*vs)[i] = scnrs[i]
+	}
+}
+
+func (vs VersionedScanners) VStoFS() []FileScanner {
+	out := make([]FileScanner, len(vs))
+	for _, vscnr := range vs {
+		out = append(out, vscnr.(FileScanner))
```

Comment:
guessing this is only called when this is guaranteed to be true?

Reply:
Hmmm, even better, I don't think the inverse methods are ever called, I'm going to delete this method (and will delete the others in a different PR once confirmed not used)